src/ProjectEnvironmentConfiger.ts

```diff
@@ -91,28 +108,143 @@ export class ProjectEnvironmentConfiger {
               telemetryContext.properties.platform = containerSelection.label;
             }
 
-            const containerItem =
-                templateJson.templates.find((template: ProjectTemplate) => {
-                  return template.name === containerSelection.label;
-                });
-
             // Configure the selected container environment for the project
-            await this.configureContainerProjectEnv(containerItem.path);
+            const res = await this.configureProjectEnv(
+                context, channel, telemetryContext, scaffoldType, templateJson,
+                rootPath, containerSelection.label);
+            if (!res) {
+              return;
+            }
+
+            // Open project in remote
+            if (openInNewWindow) {
+              setTimeout(
+                  () => vscode.commands.executeCommand(
+                      'iotcube.openInContainer', rootPath),
+                  500);
+            }
           } else {
             throw new Error(`Unsupported Platform type!`);
           }
 
-          return;
+          const message =
+              'Configuration is done. You can run \'Azure IoT Device Workbench: Compile Device Code\' command to compile device code';
+          utils.channelShowAndAppendLine(channel, message);
+          vscode.window.showInformationMessage(message);
         });
+
+    return;
   }
 
-  private async configureWorkspaceProjectEnv() {
-    // TODO
+  /**
+   * Configure Arduino project: Add tasks.json file
+   * Ask to overwrite if file already exists.
+   * @returns true - configuration success. false - configuration cancel.
+   */
+  private async configureProjectEnv(
```

Comment:
Seems this function is to configure project environment for all platform type, NOT just for 'Arduino'.
If so, please correct the comment.